test/integration/integration-test.js

```diff
@@ -147,10 +175,145 @@ describe.skip('integration tests', function() {
     ], () => done());
   });
 
+
+  it('isconnected', function(done) {
+    assert(this.api.isConnected());
+    done();
+  });
+
+
+  it('serverinfo', function(done) {
+    this.api.getServerInfo(composeAsync(data => {
+      assert(data && data.info && data.info.pubkey_node);
+    }, done));
+  });
+
+
+  it('getfee', function(done) {
+    const fee = this.api.getFee();
+    assert.strictEqual(typeof fee, 'string');
+    done();
+  });
+
+
+  it('getledgerversion', function(done) {
+    const ledgerVersion = this.api.getLedgerVersion();
+    assert.strictEqual(typeof ledgerVersion, 'number');
```

Comment:
Could also check that it's greater than a fixed constant set to the current ledger version.